libtenzir/src/format/arrow.cpp

```diff
@@ -109,6 +111,7 @@ arrow_istream_wrapper::Read(int64_t nbytes) {
 }
 
 arrow_fd_wrapper::arrow_fd_wrapper(int fd) : fd_{fd} {
+  set_mode(::arrow::io::FileMode::READ);
```

Comment:
Not sure if we really need this.

Reply:
We noticed that `READ` is the default value for the mode anyways